zerver/migrations/0121_text_choice_in_emojiset.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations, models
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+# change emojiset to text if emoji_alt_code is true.
+def change_emojiset(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    UserProfile = apps.get_model("zerver", "UserProfile")
+    for user in UserProfile.objects.filter(emoji_alt_code=True):
+        user.emojiset = "text"
+        user.save(update_fields=["emojiset"])
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0120_botuserconfigdata'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='userprofile',
+            name='emojiset',
+            field=models.CharField(choices=[('google', 'Google'), ('apple', 'Apple'), ('twitter', 'Twitter'), ('emojione', 'Emoji One'), ('text', 'Text')], default='google', max_length=20),
+        ),
```

Comment:
Now that we are storing the user choice in `emojiset` field we should drop the `emoji_alt_code` field.